Add docker environment setup instructions (#201)
SPANDigital/presidium: content/_prerequisites/02-environment-setup/03-docker.md

    ```bash
    git config --global user.name "Your Name"
    git config --global user.email "your.email@address"
    ```
    **Note:** If a Git Credential Manager (e.g. Github Desktop) is configured on the local environment, this step can be skipped.

4. Inside VS Code install the extension: [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers).

## Setting up your project
1. Copy the link to the Github repository to be cloned.

1. In [VS Code](https://code.visualstudio.com/) open the command pallet (F1 or &#8984;&#8679;`P`) and type 
`>Remote-Containers: Clone Repository in Container Volume...`
    
    **Note**: This step replaces the first step of the [Getting Started](/getting-started/#download-the-template) instructions.

1. Paste the URL for the repository which should be cloned.

1. A popup asking which branch to clone may appear. Select the one you want to checkout initally, once the container is created you are free to checkout any branch in the repository.

1. When prompted for a [Volume](https://docs.docker.com/storage/volumes/) to use there are a few options:

>     - (Recommended) Create a single volume for all project. This is the most space efficient, and unless there is a reason two project should not share a filesystem, there is no harm to this for most development environments.

Reviewer: Create a single volume for all projects

Reviewer: and unless there is a reason two projects

Reviewer: there is no harm **in** this

Author: ```suggestion
    - (Recommended) Create a single volume for all projects. This is the most space efficient, and unless there is a reason two projects should not share a filesystem, there is no harm in this for most development environments.
```

    - Create a Persistent volume for every/related projects. Share a filesystem when needed and isolate them otherwise.

>     - Create a unique volume per project. This is the least space efficient and means that no two projects can (easily) access each others files.

Reviewer: other**'s**

Author: ```suggestion
    - Create a unique volume per project. This is the least space efficient and means that no two projects can (easily) access each other's files.
```

>     The name you choose for your volume is entirely up to you and should have no impact on the project itself.

Reviewer: It is up to the developer to choose a name for their volume, and it should not have an impact on the project itself. 

Try to avoid the use of You.

Author: For some-reason I switched from Christ person half way through this and then had to go back and reword it all. Seems I missed some 🙃 

```suggestion
    It is up to the developer to choose a name for their volume, this will not have an impact on the project itself.
```

Author: For some-reason I switched from third person half way through this and then had to go back and reword it all. Seems I missed some 🙃 

```suggestion
    It is up to the developer to choose a name for their volume, this will not have an impact on the project itself.
```

1. A prompt asking for a project name will pop-up, the pre-generated name is pulled from the github repository name, and is usually a good choice.

1. The container will start building, this could take a few minutes the first time as it is downloading the required images.

> 1. A popup asking to "Add Development Configuration Files" may appear, select Jekyll (Community). You may need to click on the "Show All Definitions..." to find it. This popup will only appear if the repository does not already contain a `'.devcontainer'` folder containing the environment configuration details.

Reviewer: The developer may need to click on the "Show All Definitions..." to find it.

Author: ```suggestion
1. A popup asking to "Add Development Configuration Files" may appear, select Jekyll (Community). The developer may need to click on the "Show All Definitions..." to find it. This popup will only appear if the repository does not already contain a `'.devcontainer'` folder containing the environment configuration details.
```
